utils/swanmon/Makefile

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
Move your source code to your own repo, please.

Reply:
Once you do this, then please create also file ``test.sh``. You can find examples in this repository. It will helps to do run testing in run testing.